modules/sql-engine/src/integrationTest/sql/types/date/date_parsing.test_ignore

```diff
@@ -1,84 +1,35 @@
 # name: test/sql/types/date/date_parsing.test
 # description: Test date parsing
 # group: [date]
+# Ignore https://issues.apache.org/jira/browse/IGNITE-14602
 # Ignore https://issues.apache.org/jira/browse/IGNITE-15619
 
 statement ok
 PRAGMA enable_verification
 
 query I
-SELECT '1992-01-01'::DATE::VARCHAR == '1992-01-01'
+SELECT '30000-01-01'::DATE::VARCHAR = '30000-01-01'
 ----
 1
 
 query I
-SELECT '1992-09-20'::DATE::VARCHAR == '1992-09-20'
```

Comment:
it's better not to remove test cases from '_ignored' tests. The idea is to have single file with all the test cases. If you want to see only cases which failed, you can use any diff tool

Reply:
Fixed, thanks.